outport/firehose/firehoseIndexer_test.go

```diff
@@ -78,6 +86,73 @@ func TestFirehoseIndexer_SaveBlock(t *testing.T) {
 		require.Equal(t, errNilOutportBlock, err)
 	})
 
+	t.Run("unknown block creator, should return error", func(t *testing.T) {
+		t.Parallel()
+
+		outportBlock := createOutportBlock()
+		outportBlock.BlockData.HeaderType = "unknown"
+
+		ioWriterCalledCt := 0
+		ioWriter := &testscommon.IoWriterStub{
+			WriteCalled: func(p []byte) (n int, err error) {
+				ioWriterCalledCt++
+				return 0, nil
+			},
+		}
+
+		container := block.NewEmptyBlockCreatorsContainer()
+		_ = container.Add(core.ShardHeaderV1, block.NewEmptyHeaderCreator())
+
+		fi, _ := NewFirehoseIndexer(ioWriter, container, protoMarshaller)
+		err := fi.SaveBlock(outportBlock)
+		require.NotNil(t, err)
+		require.Equal(t, 0, ioWriterCalledCt)
+	})
+
+	t.Run("cannot marshal/unmarshall, should return error", func(t *testing.T) {
+		t.Parallel()
+
+		ioWriterCalledCt := 0
+		ioWriter := &testscommon.IoWriterStub{
+			WriteCalled: func(p []byte) (n int, err error) {
+				ioWriterCalledCt++
+				return 0, nil
+			},
+		}
+
+		unmarshallCalled := false
```

Comment:
in this file: marshall -> marshal, unmarshall -> unmarshal

Reply:
Right, fixed it